data_format.md

```diff
@@ -109,19 +115,31 @@ SCENE_DTYPE = [
 ```
 
 ### Frames
-A frame consists of information about the ego vehicle (e.g. where it was at that time), a timestamp, and a reference to the agents in that frame. Because there may be multiple agents observed we store the start and the end index.
+A frame captures all information that was observed at a time. This includes
+
+- the timestamp, which the frame describes;
+- data about the ego vehicle itself such as rotation and position;
+- a reference to the other agents (vehicles, cyclists and pedestrians) that were captured by the ego's sensors;
+- a reference to all traffic light faces for all visible lanes.
+
+The properties for both agents and traffic light faces are stored in their two respective arrays. 
+The frame contains only pointers to these stored objects in terms of a start and an end index to these arrays (again, start in included while end excluded).
```

Comment:
Here's somehow twice pointer TO - maybe change to 'The frame contains only pointers to these stored objects in terms of a start and an end index in these arrays' or just remove 'to these arrays.'

Reply:
Start IS included?